.github/workflows/test.yml

```diff
@@ -4,22 +4,14 @@ on:
   pull_request:
   push:
     branches: [main]
-  schedule:
-    - cron: '0 0 * * *'
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 jobs:
-  build:
-    name: Unit tests
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
I think it may be worth keeping around the windows testing, there are some odd cases that come up (often around import paths) that could be worth keeping an eye on.

Reply:
Yes, but I thought we could introduce it later for the e2e tests. What I deleted here was really silly: it was just eslint that was running on Mac, Linux, and Windows.

Reply:
This workflow is now also running unit tests... but there aren't any. 🤷🏼‍♂️